avocado-plugins-vt.spec

```diff
@@ -16,6 +16,9 @@ BuildRequires: python2-devel, python-setuptools
 BuildArch: noarch
 Requires: python-avocado >= 51.0
 Requires: python, autotest-framework, xz, tcpdump, iproute, iputils, gcc, glibc-headers, python-devel, nc, python-aexpect, git, python-netaddr, python-netifaces, python-simplejson
+Requires: attr # provides /usr/bin/getfattr
+Requires: policycoreutils-python # provides /usr/sbin/semanage
```

Comment:
On my F26 box, this is part of a different package (`-utils`):

```
$ rpm -qf /usr/sbin/semanage
policycoreutils-python-utils-2.6-6.fc26.x86_64
```

We'll probably need a conditional here, as I bet this is the right package name on earlier distros.  Also, please remove the comments, we can refer to version control for the "why".

Thanks!

Reply:
I've updated and pushed.